packages/devtools_app/lib/src/screens/app_size/app_size_controller.dart

```diff
@@ -229,8 +236,14 @@ class AppSizeController {
 
     changeAnalysisJsonFile(jsonFile);
 
-    // Set name for root node.
-    processedJson['n'] = 'Root';
+    // Set deferred app flag and root name if deferred app and support is on.
+    if (processedJson['n'] == 'Dummy Root' && deferredLoadingSupport) {
```

Comment:
Rename "Dummy Root" to "ArtificialRoot" (this is a better name for go/inclusive-code purposes)